testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/AccountLinkTest.java

```diff
@@ -60,27 +59,27 @@
 import static org.keycloak.testsuite.admin.ApiUtil.createUserWithAdminClient;
 
 import org.keycloak.testsuite.runonserver.RunOnServer;
+import org.keycloak.testsuite.util.AccountHelper;
 import org.keycloak.testsuite.util.FederatedIdentityBuilder;
 
 /**
  * @author <a href="mailto:bill@burkecentral.com">Bill Burke</a>
  * @version $Revision: 1 $
  */
-@DisableFeature(value = Profile.Feature.ACCOUNT2, skipRestart = true) // TODO remove this (KEYCLOAK-16228)
 public class AccountLinkTest extends AbstractKeycloakTest {
```

Comment:
@wojnarfilip @miquelsi @mhajas From my point of view this class can be deleted completely. Because it tests account console from the UI perspective and it doesn't make sense to have it more.

Reply:
already discussed it tests UserStorage provider so it makes sense to leave this test class as it is